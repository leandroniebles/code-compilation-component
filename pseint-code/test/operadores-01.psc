Proceso sin_titulo
	// estos deberian dar verdadero
	Escribir 1<2;
	Escribir 1<=2;
	Escribir 2<=2;
	Escribir 2>1;
	Escribir 2>=1;
	Escribir 2>=2;
	Escribir 2=2;
	Escribir 2==2;
	Escribir 1<>2;
	Escribir 2<>1;
	Escribir 1!=2;
	Escribir 2!=1;
	// estos deberian dar falso
	Escribir 2<1;
	Escribir 2<2;
	Escribir 2<=1;
	Escribir 1>2;
	Escribir 2>2;
	Escribir 1>=2;
	Escribir 1=2;
	Escribir 2=1;
	Escribir 1==2;
	Escribir 2==1;
	Escribir 2<>2;
	Escribir 2!=2;
FinProceso
