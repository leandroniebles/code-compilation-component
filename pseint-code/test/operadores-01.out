VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
